### PR TITLE
Conneted transaction through websocket to client

### PR DESCRIPTION
**Pull request by AightA**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -24,14 +24,23 @@ app.get("/healthcheck", (req, res) => {
...
-	  socket.on("disconnect", () => {
+	socket.on("disconnect", () => {
 		console.log("User disconnected");
-	});
+	});	
+});
+
+app.post('/api/transaction', (req, res) => {
+	api.callShopify().then( data => {
+		setInterval(() => {
```
Comment (djgrant) [inline server/server.js]: Why do you need setInterval here?

File: server/api/index.js [hunk 2]
```diff
@@ -1,8 +1,14 @@
 const Shopify = require('shopify-api-node');
 const express = require('express');
 const router = express.Router();
```
Comment (djgrant) [inline server/api/index.js]: Now that you've moved the route out of this file, `router` is unused. Also, does the file name make sense now?

File: server/server.js [hunk 3]
```diff
@@ -24,14 +24,23 @@ app.get("/healthcheck", (req, res) => {
 	res.sendStatus(200) 
 });
 
+let customeSocket = null;
```
Comment (djgrant) [inline server/server.js]: I think you mean `customerSocket`?

File: server/server.js (hunk 1 above)
Comment (AightA) [reply server/server.js]: It has been deleted.

**Pull request merged**